page-components/links.md

```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
   </ul>
 </div>
 
+## Behavior
+
+### Default links
+In general, links should default to opening in the same page or browser tab.
+
+### Opening a link in a new tab
+Links should open in a new tab in interactions where users enter data or make
```

Comment:
"in interactions" is somewhat awkward phrasing. "in situations", perhaps?

Reply:
sounds good, switched